cosmos/nois.json

```
@@ -0,0 +1,43 @@
+{
+  "rpc": "https://rpc-nois.sr20de.xyz/",
+  "rest": "https://api.noismain.mcbnode.online",
```

Should we add the proxy endpoints here?

```suggestion
  "rpc": "https://rpc.cosmos.directory/nois",
  "rest": "https://rest.cosmos.directory/nois",
```